src/05-derivatives/01-introduction.md

```diff
@@ -0,0 +1,217 @@
+# Goals and scope
+
+Derivatives are outputs of common processing pipelines, capturing data and
+meta-data sufficient for a researcher to understand and (critically) reuse those
+outputs in subsequent processing. Standardizing derivatives is motivated by use
+cases where formalized machine-readable access to processed data enables higher
+level processing.
+
+## Storage of derived datasets
+
+Derivatives can be stored/distributed in two ways:
+
+1.  Under a `derivatives/` subfolder in the root of the source BIDS dataset
+    folder to make a clear distinction between raw data and results of data
+    processing. Each pipeline has a dedicated directory under which it stores
+    all of its outputs. There are few restriction on the directory name; it is
+    RECOMMENDED to use the format `<pipeline>-<variant>` in cases where it is
+    anticipated that the same pipeline will output more than one variant (e.g.,
+    `AFNI-blurring`, `AFNI-noblurring`, etc.). For the sake of consistency, the
+    subfolder name needs to be a substring of `PipelineDescription.Name` field
+    in the `dataset_description.json` (see below).
+
+    For example:
+
+    ```Text
+    <dataset>/derivatives/fmripreprocess-v1/sub-0001
+    <dataset>/derivatives/spm/sub-0001
+    <dataset>/derivatives/vbm/sub-0001
+    ```
+
+1.  As a standalone dataset independent of the source (raw or derived) BIDS
+    dataset. This way of specifying derivatives is particularly useful when the
+    source dataset is provided with read-only access, and for publishing
+    derivatives as independent bodies of work, or for describing derivatives
+    that were created from more than one source dataset. It is consistent with
+    BIDS principles for the `sourcedata/` subdirectory to be used to include or
+    reference the source dataset(s) as it existed when the derivatives were
+    generated. Likewise, any code used to generate the derivatives from the
```

Comment:
> It is consistent with
    BIDS principles for the `sourcedata/` subdirectory to be used to include or
    reference the source dataset(s) as it existed when the derivatives were
    generated.

This sentence is not very readable, please rephrase.

Reply:
Please see updated text.